src/views/components/attendees-list/attendees.php

```diff
@@ -26,6 +26,10 @@
 	return;
 }
 
+// Sort the Attendees by their ID.
+$attendee_ids = array_column($attendees, 'ID');
```

Comment:
Can we please move this logic to the [call to the template](https://github.com/the-events-calendar/event-tickets/blob/master/src/Tickets/Commerce/Shortcodes/Success_Shortcode.php#L50)? We should avoid having logic in the templates. Also, please take a look at the spacing between the parameters and the parenthesis.